packages/turf-interpolate/index.js

```diff
@@ -6,10 +6,20 @@ import { centroid } from "@turf/centroid";
 import { squareGrid } from "@turf/square-grid";
 import { triangleGrid } from "@turf/triangle-grid";
 import { clone } from "@turf/clone";
-import { featureCollection } from "@turf/helpers";
+import { featureCollection, validateBBox } from "@turf/helpers";
 import { featureEach } from "@turf/meta";
 import { collectionOf } from "@turf/invariant";
 
+function checkBBox(bbox) {
```

Comment:
Is this providing extra protection over and above just calling validateBBox? Also, why the check for `bbox.bbox`?

Reply:
Yes, I overthought it. This check is redundant since the validateBBox method already handles the validation、